pages/tutorials/developing/visualize-engagement-metrics-grafana.en-US.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: 'How to Visualize Your Engagement Metrics With Grafana'
+description: 'Learn how to quickly standup a Grafana dashboard with the core engagement and performance metrics of your video platform'
+type: 'tutorial'
+category: 'developing'
+ogImageText: 'How to Visualize Your Engagement Metrics With Grafana'
+---
+
+import BlogHeader from '@components/core/Tutorial/Header';
+
+<BlogHeader />
+
+In May 2023, we released powerful engagement metrics offering detailed information on viewer behavior and playback quality on your platform.
+The API includes engagement metrics such as view counts and watch time, as well as performance metrics such as error rate, time to first frame, rebuffer ratio, and exit-before-starts across a variety of dimensions.
+For more details, check out the docs [here](/guides/developing/viewer-engagement).
+
+In this tutorial we will introduce a simple, free, and customizable method to quickly visualize the core metrics and dimensions of the API.
+
+## Prerequisites
+
+Before you begin this tutorial, make sure you have:
+
+1. Integrated the necessary components to capture viewership data. Detail [here](/guides/developing/viewer-engagement#registering-views).
+2. Viewed videos via (1) to collect data.
+3. Created an CORS-enabled API Key [here](https://livepeer.com/dashboard/developers/api-keys). The key must allow CORS access from your Grafana origin, or from all (\*)
+4. Set up a Grafana account and workspace. A free account is sufficient and available at [Grafana.com](https://grafana.com/)
+
+With that we are ready to set up our dashboard!
+
+## Setting up the Engagement Dashboard
+
+1. Login to Grafana
+2. Click “Connections” > "Connect Data"
+   <img src="/images/tutorials/grafana-connections.jpg" alt="" width="30%" />
```

Comment:
This image is missing

Reply:
@0xcadams  it works locally. what would cause it to work locally but in vercel?